src/solver.cpp

```diff
@@ -54,7 +54,7 @@ int Solver::run(const realtype tout, ReturnData *rdata) const {
     } else {
         status = solve(tout, AMICI_NORMAL);
     }
-    rdata->cpu_time +=(double)((clock() - starttime) * 1000) / CLOCKS_PER_SEC;
+    incrementCpuTime((realtype)((clock() - starttime) * 1000) / CLOCKS_PER_SEC);
```

Comment:
its a private member, you should still be able to modify it without a setter routine. I don't think we want a setter routine as the `Solver` class itself should be the only one that modifies those values.

Reply:
okay, I'll do that. Thanks!
Other question: How about the way, it's currently implemented in forward or backwardproblem.cpp? Would you prefer incrementing the rdata-member after every run, or once at the very end of the routine (as it is now)?

Reply:
> Other question: How about the way, it's currently implemented in forward or backwardproblem.cpp? Would you prefer incrementing the rdata-member after every run, or once at the very end of the routine (as it is now)?

It's fine as it is now, as it is consistent with how other things are handled at the moment. In the long run I want to change this to be all done in a single function that can be called independently of forward/backwardproblem, but one thing at a time.